packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -119,6 +119,13 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
     }
   }
 
+  /**
+   * Returns a flag indicating whether the search box has focus
+   */
+  public hasFocus(): boolean {
+    return !!this.state.hasFocus;
```

Comment:
I think `this.state.hasFocus` should be sufficient since it's a boolean.

Reply:
But it may be undefined